src/compile/inference/imports.ts

```diff
@@ -137,10 +159,8 @@ export function findAllFiles(
 
         for (const fld of flds) {
             if (fld.kind === FileLevelNodeKind.Import) {
-                queue.push([realPath, fld.path]);
+                queue.push(computeSourceUnitName(sourceUnitName, fld.path, remappings));
```

Comment:
**This is a good call**. I didn't figured it out during the last optimization pass and decided to preserve importer/imported paths as separate entities. But now I see that it caused way too much complications in codebase above.